drivers/sensors/apds9922.c

```diff
@@ -0,0 +1,2598 @@
+static int apds9922_als_variance(FAR struct apds9922_dev_s *priv,
+                                     enum apds9922_als_thresh_var variance);
+static int apds9922_als_thresh(FAR struct apds9922_dev_s *priv,
+                               FAR struct adps9922_als_thresh thresholds);
+static int apds9922_als_int_mode(FAR struct apds9922_dev_s *priv,
+                                 enum apds9922_als_int_mode mode);
+static int apds9922_als_channel(FAR struct apds9922_dev_s *priv,
+                                enum apds9922_als_chan channel);
+static int apds9922_als_factor(FAR struct apds9922_dev_s *priv,
+                               uint32_t factor);
+static int apds9922_als_limit(FAR struct apds9922_dev_s *priv,
+                               uint32_t limit);
+
+/* Proximity sensor functions */
+
+static int apds9922_ps_config(FAR struct apds9922_dev_s *priv,
+                             FAR struct apds9922_ps_setup_s *config);
+static int apds9922_ps_resolution(FAR struct apds9922_dev_s *priv,
+                                  enum apds9922_ps_res res);
+static int apds9922_ps_rate(FAR struct apds9922_dev_s *priv,
+                            enum apds9922_ps_rate rate);
+static int apds9922_ps_ledf(FAR struct apds9922_dev_s *priv,
+                            enum apds9922_ps_led_f freq);
+static int apds9922_ps_ledi(FAR struct apds9922_dev_s *priv,
+                            enum apds9922_ps_led_i current);
+static int apds9922_ps_ledpk(FAR struct apds9922_dev_s *priv, bool enable);
+static int apds9922_ps_pulses(FAR struct apds9922_dev_s *priv,
+                              uint8_t num_p);
+static int apds9922_ps_thresh(FAR struct apds9922_dev_s *priv,
+                              FAR struct adps9922_ps_thresh thresh);
+static int apds9922_ps_canc_lev(FAR struct apds9922_dev_s *priv,
+                                uint16_t lev);
+static int apds9922_ps_int_mode(FAR struct apds9922_dev_s *priv,
+                                enum apds9922_ps_int_mode mode);
+static int apds9922_ps_persistance(FAR struct apds9922_dev_s *priv,
+                                   uint8_t persistance);
+static int apds9922_ps_notify_mode(FAR struct apds9922_dev_s *priv,
+                                   enum apds9922_ps_notify);
+
+/* Character driver methods */
+
+static int     apds9922_open(FAR struct file *filep);
+static int     apds9922_close(FAR struct file *filep);
+static ssize_t apds9922_als_read(FAR struct file *, FAR char *, size_t);
+static ssize_t apds9922_als_write(FAR struct file *filep,
+                                  FAR const char *buffer, size_t buflen);
+static int     apds9922_als_ioctl(FAR struct file *filep, int cmd,
+                                  unsigned long arg);
+static int     apds9922_als_poll(FAR struct file *filep,
+                                 FAR struct pollfd *fds, bool setup);
+static ssize_t apds9922_ps_read(FAR struct file *, FAR char *, size_t);
```

Comment:
Please use the same style as in other places

Reply:
The missing filep I guess, two place? Now corrected that.

Reply:
Yes. Other places have named parameres

Reply:
Done